### PR TITLE
Support for Term Splitting in WP 4.2

### PR DESCRIPTION
**Pull request by sc0ttkclark**
This code addresses #2660

### CODE REVIEW

File: classes/PodsTermSplitting.php [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+			$wpdb->update( $table, $data, $where, $format, $where_format );
+
+			$this->append_progress( $task );
+		}
+
+		// Fix up the serialized data
+		$task = "update_setting_meta_{$pod_name}_{$field_name}_serialized";
+		if ( ! $this->have_done( $task ) ) {
+
+			$target_serialized = sprintf( ';i:%s;', $this->term_id );
```
Comment (sc0ttkclark) [inline classes/PodsTermSplitting.php]: These probably should have been `%d`

Comment (pglewis) [inline classes/PodsTermSplitting.php]: No, the fields are string fields and need quoted, string values.

Comment (sc0ttkclark) [inline classes/PodsTermSplitting.php]: But `;i:%s;` is saying integer, so it would be `;i:%d;`

Comment (pglewis) [inline classes/PodsTermSplitting.php]: Integer is how it's to be interpreted when unserialized.  It is still stored as a string in the database and thus we need to build a string.

**Pull request merged**